src/main/java/com/box/sdk/BoxTask.java

```diff
@@ -0,0 +1,281 @@
+            BoxTaskAssignment.Info info = assignment.new Info(assignmentJSON);
+            assignments.add(info);
+        }
+
+        return assignments;
+    }
+
+    /**
+     * Gets information about this task.
+     * @return info about this task.
+     */
+    public Info getInfo() {
+        URL url = TASK_URL_TEMPLATE.build(this.getAPI().getBaseURL(), this.getID());
+        BoxAPIRequest request = new BoxAPIRequest(this.getAPI(), url, "GET");
+        BoxJSONResponse response = (BoxJSONResponse) request.send();
+        JsonObject responseJSON = JsonObject.readFrom(response.getJSON());
+        return new Info(responseJSON);
+    }
+
+    /**
+     * Updates the information about this task with any info fields that have been modified locally.
+     *
+     * <p>The only fields that will be updated are the ones that have been modified locally. For example, the following
+     * code won't update any information (or even send a network request) since none of the info's fields were
+     * changed:</p>
+     *
+     * <pre>BoxTask task = new BoxTask(api, id);
+     *BoxTask.Info info = task.getInfo();
+     *task.updateInfo(info);</pre>
+     *
+     * @param info the updated info.
+     */
+    public void updateInfo(BoxTask.Info info) {
+        URL url = TASK_URL_TEMPLATE.build(this.getAPI().getBaseURL(), this.getID());
+        BoxJSONRequest request = new BoxJSONRequest(this.getAPI(), url, "PUT");
+        request.setBody(info.getPendingChanges());
+        BoxJSONResponse response = (BoxJSONResponse) request.send();
+        JsonObject jsonObject = JsonObject.readFrom(response.getJSON());
+        info.update(jsonObject);
+    }
+
+    /**
+     * Contains information about a BoxTask.
+     */
+    public class Info extends BoxResource.Info {
+        private BoxFile.Info item;
+        private Date dueAt;
+        private String action;
+        private String message;
+        private List<BoxTaskAssignment.Info> taskAssignments;
+        private boolean completed;
+        private BoxUser.Info createdBy;
+        private Date createdAt;
+
+        /**
+         * Constructs an empty Info object.
+         */
+        public Info() {
+            super();
+        }
+
+        /**
+         * Constructs an Info object by parsing information from a JSON string.
+         * @param  json the JSON string to parse.
+         */
+        public Info(String json) {
+            super(json);
+        }
+
+        /**
+         * Constructs an Info object using an already parsed JSON object.
+         * @param  jsonObject the parsed JSON object.
+         */
+        Info(JsonObject jsonObject) {
+            super(jsonObject);
+        }
+
+        @Override
+        public BoxResource getResource() {
+            return BoxTask.this;
+        }
+
+        /**
+         * Gets the file associated with this task.
+         * @return the file associated with this task
```

Comment:
Please put period after the return description. True for multiple comments in this file.